content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -93,6 +93,7 @@ different Kubernetes components.
 | `TokenRequest` | `false` | Alpha | 1.10 | |
 | `VolumeScheduling` | `false` | Alpha | 1.9 | 1.9 |
 | `VolumeScheduling` | `true` | Beta | 1.10 | |
+| `ScheduleDaemonSetPods` | `false` | Alpha | 1.11 | |
```

Comment:
- version incorrect
- mind the alpha-beta order
- provide a short description for the gate